client/a8c-for-agencies/sections/team/hooks/use-handle-member-action.ts

```diff
@@ -14,12 +15,42 @@ export default function useHandleMemberAction( { onRefetchList }: Props ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 
+	const { mutate: resendMemberInvite } = useResendMemberInviteMutation();
+
 	const { mutate: cancelMemberInvite } = useCancelMemberInviteMutation();
 
 	const { mutate: removeMember } = useRemoveMemberMutation();
 
 	return useCallback(
 		( action: string, item: TeamMember, callback?: () => void ) => {
+			if ( action === 'resend-user-invite' ) {
+				resendMemberInvite(
+					{ id: item.id },
+					{
+						onSuccess: () => {
+							dispatch(
+								successNotice( translate( 'The invitation has been resend.' ), {
```

Comment:
Minor grammar issue: The invitation has been ~resend~ **resent**.

Reply:
Thanks, this has been fixed. 👍